# joint_plan.yaml

joint_test_client:
  goal_configuration: [-1.0, 0.0, 0.0, 0.2, 1.3]
  initial_configuration: [2.0, 1.0, 0.0, 0.2, 1.3, 1.57]
  planning_group: "manipulator"
